

from PyQt5 import QtCore, QtGui, QtWidgets
# ไม่ใช้แล้ว

class Ui_each_drug2(object):
    def setupUi(self, each_drug2):
        each_drug2.setObjectName("each_drug2")
        each_drug2.resize(531, 401)
        each_drug2.setStyleSheet("\n"
"background-color: rgb(23, 73, 110);")
        self.centralwidget = QtWidgets.QWidget(each_drug2)
        self.centralwidget.setObjectName("centralwidget")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setGeometry(QtCore.QRect(0, -60, 531, 131))
        self.frame.setStyleSheet("border-radius: 40px;\n"
"background-color: rgb(255, 255, 255);")
        self.frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.label = QtWidgets.QLabel(self.frame)
        self.label.setGeometry(QtCore.QRect(180, 70, 171, 51))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setStyleSheet("border-radius: 16px;\n"
"color: rgb(255, 255, 255);\n"
"background-color: rgb(23, 73, 110);")
        self.label.setFrameShape(QtWidgets.QFrame.Box)
        self.label.setLineWidth(1)
        self.label.setTextFormat(QtCore.Qt.AutoText)
        self.label.setScaledContents(False)
        self.label.setAlignment(QtCore.Qt.AlignCenter)
        self.label.setWordWrap(True)
        self.label.setObjectName("label")
        self.add_back_pushButton = QtWidgets.QPushButton(self.frame)
        self.add_back_pushButton.setGeometry(QtCore.QRect(50, 80, 71, 31))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.add_back_pushButton.setFont(font)
        self.add_back_pushButton.setStyleSheet("border-radius: 9px;\n"
"color: rgb(0, 0, 0);\n"
"background-color: rgb(244, 212, 99);")
        self.add_back_pushButton.setObjectName("add_back_pushButton")
        self.delete_pushButton = QtWidgets.QPushButton(self.frame)
        self.delete_pushButton.setGeometry(QtCore.QRect(420, 80, 41, 31))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.delete_pushButton.setFont(font)
        self.delete_pushButton.setStyleSheet("border-radius: 9px;\n"
"background-color: rgb(171, 57, 57);\n"
"color: rgb(255, 255, 255);\n"
"")
        self.delete_pushButton.setObjectName("delete_pushButton")
        self.frame_2 = QtWidgets.QFrame(self.centralwidget)
        self.frame_2.setGeometry(QtCore.QRect(20, 90, 231, 211))
        self.frame_2.setStyleSheet("border-radius: 16px;\n"
"background-color: rgb(236, 236, 236);")
        self.frame_2.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_2.setObjectName("frame_2")
        self.drugSize = QtWidgets.QLabel(self.frame_2)
        self.drugSize.setGeometry(QtCore.QRect(10, 20, 191, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.drugSize.setFont(font)
        self.drugSize.setObjectName("drugSize")
        self.drugNew = QtWidgets.QLabel(self.frame_2)
        self.drugNew.setGeometry(QtCore.QRect(10, 110, 200, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.drugNew.setFont(font)
        self.drugNew.setObjectName("drugNew")
        self.size_label = QtWidgets.QLabel(self.frame_2)
        self.size_label.setGeometry(QtCore.QRect(10, 50, 211, 41))
        font = QtGui.QFont()
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.size_label.setFont(font)
        self.size_label.setStyleSheet("border-radius: 9px;\n"
"color: rgb(0, 0, 0);\n"
"background-color: rgb(255, 255, 255);")
        self.size_label.setFrameShape(QtWidgets.QFrame.Box)
        self.size_label.setLineWidth(1)
        self.size_label.setTextFormat(QtCore.Qt.AutoText)
        self.size_label.setScaledContents(False)
        self.size_label.setAlignment(QtCore.Qt.AlignCenter)
        self.size_label.setWordWrap(True)
        self.size_label.setObjectName("size_label")
        self.label_6 = QtWidgets.QLabel(self.frame_2)
        self.label_6.setGeometry(QtCore.QRect(10, 140, 211, 41))
        font = QtGui.QFont()
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_6.setFont(font)
        self.label_6.setStyleSheet("border-radius: 9px;\n"
"color: rgb(0, 0, 0);\n"
"background-color: rgb(255, 255, 255);")
        self.label_6.setFrameShape(QtWidgets.QFrame.Box)
        self.label_6.setLineWidth(1)
        self.label_6.setTextFormat(QtCore.Qt.AutoText)
        self.label_6.setScaledContents(False)
        self.label_6.setAlignment(QtCore.Qt.AlignCenter)
        self.label_6.setWordWrap(True)
        self.label_6.setObjectName("label_6")
        self.frame_3 = QtWidgets.QFrame(self.centralwidget)
        self.frame_3.setGeometry(QtCore.QRect(270, 90, 241, 211))
        self.frame_3.setStyleSheet("border-radius: 16px;\n"
"background-color: rgb(236, 236, 236);")
        self.frame_3.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_3.setObjectName("frame_3")
        self.drugStill = QtWidgets.QLabel(self.frame_3)
        self.drugStill.setGeometry(QtCore.QRect(-30, 20, 251, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.drugStill.setFont(font)
        self.drugStill.setAlignment(QtCore.Qt.AlignCenter)
        self.drugStill.setObjectName("drugStill")
        self.drugGot = QtWidgets.QLabel(self.frame_3)
        self.drugGot.setGeometry(QtCore.QRect(-20, 110, 261, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.drugGot.setFont(font)
        self.drugGot.setAlignment(QtCore.Qt.AlignCenter)
        self.drugGot.setObjectName("drugGot")
        self.label_7 = QtWidgets.QLabel(self.frame_3)
        self.label_7.setGeometry(QtCore.QRect(10, 50, 221, 41))
        font = QtGui.QFont()
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_7.setFont(font)
        self.label_7.setStyleSheet("border-radius: 9px;\n"
"color: rgb(0, 0, 0);\n"
"background-color: rgb(255, 255, 255);")
        self.label_7.setFrameShape(QtWidgets.QFrame.Box)
        self.label_7.setLineWidth(1)
        self.label_7.setTextFormat(QtCore.Qt.AutoText)
        self.label_7.setScaledContents(False)
        self.label_7.setAlignment(QtCore.Qt.AlignCenter)
        self.label_7.setWordWrap(True)
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(self.frame_3)
        self.label_8.setGeometry(QtCore.QRect(10, 140, 221, 41))
        font = QtGui.QFont()
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.label_8.setFont(font)
        self.label_8.setStyleSheet("border-radius: 9px;\n"
"color: rgb(0, 0, 0);\n"
"background-color: rgb(255, 255, 255);")
        self.label_8.setFrameShape(QtWidgets.QFrame.Box)
        self.label_8.setLineWidth(1)
        self.label_8.setTextFormat(QtCore.Qt.AutoText)
        self.label_8.setScaledContents(False)
        self.label_8.setAlignment(QtCore.Qt.AlignCenter)
        self.label_8.setWordWrap(True)
        self.label_8.setObjectName("label_8")
        self.next_pushButton = QtWidgets.QPushButton(self.centralwidget)
        self.next_pushButton.setGeometry(QtCore.QRect(430, 330, 71, 31))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.next_pushButton.setFont(font)
        self.next_pushButton.setStyleSheet("border-radius: 9px;\n"
"color: rgb(0, 0, 0);\n"
"background-color: rgb(227, 151, 61);")
        self.next_pushButton.setObjectName("next_pushButton")
        each_drug2.setCentralWidget(self.centralwidget)

        self.retranslateUi(each_drug2)
        QtCore.QMetaObject.connectSlotsByName(each_drug2)

    def retranslateUi(self, each_drug2):
        _translate = QtCore.QCoreApplication.translate
        each_drug2.setWindowTitle(_translate("each_drug2", "ยาแต่ละตัว"))
        self.label.setText(_translate("each_drug2", "ชื่อยา"))
        self.add_back_pushButton.setText(_translate("each_drug2", "ย้อนกลับ"))
        self.delete_pushButton.setText(_translate("each_drug2", "ลบ"))
        self.drugSize.setText(_translate("each_drug2", "ขนาดเม็ดยา (มิลลิเมตร)"))
        self.drugNew.setText(_translate("each_drug2", "จำนวนยาที่ได้รับมาใหม่ (เม็ด)"))
        self.size_label.setText(_translate("each_drug2", "ขนาดเม็ดยา"))
        self.label_6.setText(_translate("each_drug2", "ยาที่ได้รับมาใหม่"))
        self.drugStill.setText(_translate("each_drug2", "จำนวนมื้อยาคงเหลือ (มื้อ)"))
        self.drugGot.setText(_translate("each_drug2", "จำนวนยาที่ได้รับมาแล้ว (เม็ด)"))
        self.label_7.setText(_translate("each_drug2", "ยาคงเหลือ"))
        self.label_8.setText(_translate("each_drug2", "ยาที่ได้รับมาแล้ว"))
        self.next_pushButton.setText(_translate("each_drug2", "ถัดไป"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    each_drug2 = QtWidgets.QMainWindow()
    ui = Ui_each_drug2()
    ui.setupUi(each_drug2)
    each_drug2.show()
    sys.exit(app.exec_())
